fennel/datasets/datasets.py

```diff
@@ -84,6 +85,7 @@
 
 DEFAULT_RETENTION = Duration("2y")
 DEFAULT_EXPIRATION = Duration("30d")
+DEFAULT_VERSION = 0
```

Comment:
I think it might be better to start versions from 1, not 0?

Reply:
makes sense, will change I wrote 0 because I remember in featureset extractor we used to start from 0.